packages/gatsby-tinacms-json/index.ts

```diff
@@ -32,9 +32,6 @@ export function useJsonForm(
   jsonNode: JsonNode,
   formOptions: Partial<FormOptions<any>> = {}
 ) {
-  if (!jsonNode || process.env.NODE_ENV === 'production') {
-    return [{}, null]
-  }
```

Comment:
This is probably the one case where it might really make sense to use an `eslint-disable`, but this raises a couple questions:

 - Why is this functionality forcibly disabled in production environments anyway?
 - Is the check for `jsonNode` actually necessary? What's the use case?

Reply:
This was intentional behaviour. It is a performance improvement that prevents useJsonForm from doing anything in production. This change should be undone.

Reply:
So should I change the type of `jsonNode` to be optional?